Installation/Windows/client/Templates/NSIS.template.in

```diff
@@ -762,7 +762,7 @@ Function check_installation_directory
    StrCmp $1 "0" handle_error
    return
    handle_error:
-     MessageBox MB_YESNO "you aren't allowed to install arango in $INSTDIR do you want try once more?" IDNO no
+     MessageBox MB_YESNO "You are not allowed to install ArangoDB in ${INSTDIR}.\n\nDo you want try once more?" IDNO no

```

Comment:
its $\n

Reply:
Is `$` used as escape character in NSIS templates?

Reply:
https://nsis-dev.github.io/NSIS-Forums/html/t-164596.html - and I think ${INSTDIR} changes the meaning in comparison to $INSTDIR too, its probably introducing an error.

Reply:
The link seems to be about MUI. I wonder if it applies to MessageBox as well.
About $INSTDIR, any idea how else one would do this for NSIS, like if it was $INSTDIRsomething, to separate between the variable name and the other text?

Reply:
`${NAME}` refers to a definition (`!define NAME 'value'`), whereas `$NAME` refers to a variable:

    var /GLOBAL NAME ; global flag required in sections and functions
    StrCpy $NAME "value"

To resolve an ambiguity between variables, e.g. `$INSTDIR` vs `$INST` + `"DIR"`, the only way I found is to do the following:

    var /GLOBAL _
    StrCpy $_ ""
    MessageBox MB_OK "$INST$_DIR"